etc/repos.yaml

```diff
@@ -470,3 +470,4 @@ display_astrowidgets: https://github.com/lsst/display_astrowidgets
 analysis_ap: https://github.com/lsst/analysis_ap
 daf_relation: https://github.com/lsst/daf_relation
 ctrl_platform_s3df: https://github.com/lsst/ctrl_platform_s3df
+meas_transiNet: https://github.com/lsst/meas_transiNet
```

Comment:
Is the plan to not put `rbClassifier_data` in here because it's going away? That would mean it will never be available to `stack-os-matrix` or other `lsstsw` builds, and that it can't be added as a dependency of `meas_transiNet`.

Reply:
Sorry, my mistake; I'd misremembered the RFC. The [final proposal](https://jira.lsstcorp.org/browse/RFC-909?focusedCommentId=2280120#comment-2280120) was that `rbClassifier_data` would _not_ be added to the stack. So no `repos.yaml`, no `setupOptional`, etc.